spec/models/job_post_spec.rb

```diff
@@ -64,5 +64,11 @@
       job_post.tag(tag1, tag2)
       expect(job_post.tags.first).to eq(tag1)
     end
+
+    it "should be commentable" do 
+      comment1 = FactoryGirl.create(:comment,commentable_type:job_post.class,commentable_id: job_post.id)
+      comment2 = FactoryGirl.create(:comment,commentable_type:job_post.class,commentable_id: job_post.id)
+      expect(job_post.comments).to eq([comment1,comment2])
+    end
   end
```

Comment:
so if you do the `transient` thing above you can create comments here like:
```
 comment1 = FactoryGirl.create(:comment, commentable: job_post)
```
and it will work exactly the same.